konig-content-system-gae/src/main/java/io/konig/content/gae/GaeContentSystemServlet.java

```diff
@@ -21,22 +23,61 @@
  */
 
 
-import java.io.IOException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.ArrayList;
+import java.util.List;
 
-import javax.servlet.ServletConfig;
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+
+import com.google.appengine.api.memcache.MemcacheService;
+import com.google.appengine.api.memcache.MemcacheServiceFactory;
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+
 
 public class GaeContentSystemServlet extends HttpServlet {
 	private static final long serialVersionUID = 1L;
-	private static final String ZIP_CONTENT_TYPE = "application/zip";
+	private static final String ZIP_CONTENT_TYPE = "application/zip";
+	public static final String ALLOWED_DOMAINS = "allowed-domains";
 	
 	private GaeCheckInBundleServlet checkInBundle = new GaeCheckInBundleServlet();
 	private GaeAssetServlet asset = new GaeAssetServlet();
 	private GaeZipBundleServlet zipBundle = new GaeZipBundleServlet();
-
+	
+	@Override
+	public void init() throws ServletException {
+		String configFile = getServletConfig().getInitParameter("configFile");
+		MemcacheService cache = MemcacheServiceFactory.getMemcacheService();
```

Comment:
Memcache is overkill here.  Let's just add a  HashSet as a static field and use that instead.  
We should only use Memcache if there is a concern about the volume of data being cached.  In this case, the volume is tiny.